### PR TITLE
Refactor and verification code

### PR DESCRIPTION
**Pull request by mayajohns**
### Requirements

code refactoring with css

### CODE REVIEW

File: core/jazz_ui/e2e/04-jazzservices/jazzservices.e2e-spec.ts [hunk 1]
```diff
@@ -48,15 +48,18 @@ describe('Overview', () => {
             browser.wait(EC.elementToBeClickable(jazzServices_po.getCreateService()), timeOutHigh);
             winhandle = browser.getWindowHandle();
             jazzServices_po.getCreateService().click();
-            servicename='servicename' + Date.now();
+            var min = 111111111;
+            var max = 999999999;
+ 	        var randomNum = Math.floor(Math.random() * (max - min + 1)) + min;
```
Comment (svidhani) [inline core/jazz_ui/e2e/04-jazzservices/jazzservices.e2e-spec.ts]: indentation

File: core/jazz_ui/e2e/04-jazzservices/jazzservices.e2e-spec.ts [hunk 2]
```diff
@@ -48,15 +48,18 @@ describe('Overview', () => {
             browser.wait(EC.elementToBeClickable(jazzServices_po.getCreateService()), timeOutHigh);
             winhandle = browser.getWindowHandle();
             jazzServices_po.getCreateService().click();
-            servicename='servicename' + Date.now();
+            var min = 111111111;
```
Comment (svidhani) [inline core/jazz_ui/e2e/04-jazzservices/jazzservices.e2e-spec.ts]: why not use something like Math.min and math.max

File: core/jazz_ui/e2e/page-objects/jazzservices.po.ts [hunk 3]
```diff
@@ -145,4 +124,8 @@ export class Jazz {
   getAssetStatusVerify(){
     return element(by.xpath('//li[@class="x caps active"]'));
   }
+  getAPIServiceName(){
+    return element(by.xpath('//div[@class="table-row pointer"]/div[text()="api"]/preceding-sibling::div'));
```
Comment (svidhani) [inline core/jazz_ui/e2e/page-objects/jazzservices.po.ts]: is this still going to be xpath?

File: core/jazz_ui/e2e/04-jazzservices/jazzservices.e2e-spec.ts (hunk 1 above)
Comment (mayajohns) [reply core/jazz_ui/e2e/04-jazzservices/jazzservices.e2e-spec.ts]: done

File: core/jazz_ui/e2e/04-jazzservices/jazzservices.e2e-spec.ts (hunk 2 above)
Comment (mayajohns) [reply core/jazz_ui/e2e/04-jazzservices/jazzservices.e2e-spec.ts]: We are using min and max for the random generation. This is just a declaration for that.
Ref:  var randomNum = Math.floor(Math.random() * (Math.max() - Math.min() + 1)) + Math.min();

File: core/jazz_ui/e2e/04-jazzservices/jazzservices.e2e-spec.ts [hunk 4]
```diff
@@ -121,67 +85,65 @@ describe('Overview', () => {
...
+      it('Create Website Service', () => {
+            browser.driver.switchTo().activeElement();
+            browser.driver.sleep(5000);
+            browser.wait(EC.visibilityOf(jazzServices_po.getCreateService()), timeOutHigh);
+            browser.wait(EC.elementToBeClickable(jazzServices_po.getCreateService()), timeOutHigh);
+            jazzServices_po.getCreateService().click();
+            browser.driver.switchTo().activeElement();
+            browser.driver.sleep(5000);
+            jazzServices_po.getWebsite().click();
+            var min = 111111111;
```
Comment (svidhani) [inline core/jazz_ui/e2e/04-jazzservices/jazzservices.e2e-spec.ts]: try to use const, let ,etc instead of var

**Pull request merged**